### ISSUE TITLE
[Steamshovel] Speed-meter for displaying events (Trac #1422)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1414]
Make the right hand side slider for the playing-speed of events be an absolute on the speed instead of in relation to the length of the displayed time-range.

Playing successive events after one other, it is very confusing that the displayed time-evolution is not a constant physical speed (X ns(detector)/s(display)).
This makes the physical processes not comparable across events.

Migrated from https://code.icecube.wisc.edu/ticket/1422
```json
{
    "status": "closed", 
    "changetime": "2016-03-18T21:14:08", 
    "description": "Make the right hand side slider for the playing-speed of events be an absolute on the speed instead of in relation to the length of the displayed time-range.\n\nPlaying successive events after one other, it is very confusing that the displayed time-evolution is not a constant physical speed (X ns(detector)/s(display)).\nThis makes the physical processes not comparable across events.\n", 
    "reporter": "mzoll", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1458335648518728", 
    "component": "combo core", 
    "summary": "[Steamshovel] Speed-meter for displaying events", 
    "priority": "normal", 
    "keywords": "steamshovel", 
    "time": "2015-11-05T13:00:58", 
    "milestone": "", 
    "owner": "hdembinski", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20151105T16:31:07: hdembinski changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20151115T01:00:43: 
* hdembinski changed status from "accepted" to "closed"
* hdembinski changed resolution from "" to "fixed"
* hdembinski commented:

> In changeset 139525:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="139525"
> animation speed is now an absolute quantity, and expressed in nanoseconds (event) per second (real time), closes #1414
> ```

**Comment by icecube-trac** [comment]
Trac update at 20160318T21:14:08: 
* olivas commented:

> Milestone December 2015 deleted


* olivas changed milestone from "December 2015" to ""
